tags/line_commands/line_commands.py

```diff
@@ -39,3 +39,9 @@ def line_clone(line: int):
         """Clones specified line at current position"""
         actions.edit.jump_line(line)
         actions.edit.line_clone()
+
+    def jump_cursor_to_next_char(char: str):
+        """Jumps the first to the next instance of the specified character on the current line"""
+
+    def jump_cursor_to_prev_char(char: str):
+        """Jumps the first to the previous instance of the specified character on the current line"""
```

Comment:
qq: could you elaborate why limiting search results to the current line is valuable? I thought it was because it would wrap so you could get to stuff closer to the beginning quickly, but it looks like in vim `f` doesn't do that. Probably something we were missing. cc @nriley who tried it

Reply:
I often fuzzily (as in I don't know the exact index so overshoot) jump to the last character on a line (and don't want to jump off the line) and also don't always want to have to speak multiple commands to go to the end of the line and then search backwards. Also helps if you are working on a single line and definitely don't want to go off that line in the event of a misrecognition/mispeak (a regularity for me).

If others don't think that's actually useful to general users happy to just close.